app/src/components/UserAddress.vue

```diff
@@ -2,15 +2,25 @@
   <b-tag rounded class="user-address">
     <span class="address" :title="address">
       <img :src="avatarUrl" class="img mr-1" />
-      <template v-if="truncate">{{ address | truncate(10) }}</template>
-      <template v-else>{{ address }}</template>
+      <a target="_blank" :href="ethersConnect.getNetExplorerUrl() + 'address/' + address">
+        <template v-if="truncate">{{ address | truncate(10) }}</template>
+        <template v-else>{{ address }}</template>
+      </a>
+      <span @click="copyAddress"><b-icon icon="clipboard" type="is-primary" /></span>
```

Comment:
Icon is not aligned and spaced properly. It also needs to be sized and colored differently. Additionally it needs to have cursor pointer.

Reply:
Where and to what will be properly aligned and spaced and what is the right size and color? The color needs to be the reverted version of the icon?

Reply:
looks good now